src/main/scala/org/akkajs/sbt/SbtClient.scala

```diff
@@ -2,12 +2,23 @@ package org.akkajs.sbt
 
 import scala.scalajs.js
 
-import scala.concurrent.{Future, Promise}
+import scala.concurrent.Future
 import scala.concurrent.ExecutionContext.Implicits.{global => ec}
 
 import com.definitelyscala.node.Node
 
-object SbtCli extends App {
+object SbtClient extends App {
+
+  /* Cherry-picked commands from 'sbt completions' */
+  val allCommands = Seq("clean",
```

Comment:
this list has to come from sbt itself, we do not have any way to know this list of commands ahead of time.

This is some WIP for getting there: https://github.com/sbt/sbt/pull/4385

Reply:
I see... Now https://github.com/sbt/sbt/pull/4397 . Lets close this pull request and wait for developments in that front.

Reply:
Correct! as soon as we have that merged we can revamp this one :-)